src/main/kotlin/racing/service/GameMachine.kt

```diff
@@ -0,0 +1,32 @@
+package racing.service
+
+import racing.model.Racing
+
+object GameMachine {
+    private const val CAR_SYMBOL = "-"
+    private const val RESULT_SENTENCE = "실행 결과"
+    fun askCounting(question: String): Int {
+        println(question)
+        return readln().orEmpty().toInt()
+    }
+
+    fun printResult(carCount: Int, tryGameCount: Int) {
+        println(RESULT_SENTENCE)
+        play(carCount, tryGameCount)
+    }
+
+    fun play(carCount: Int, tryGameCount: Int) {
+        val racing = Racing(carCount)
+        repeat(tryGameCount) {
+            racing.cars.forEach { car ->
+                val moving = car.run()
+                repeat(moving) {
+                    print(CAR_SYMBOL)
+                }
+                println()
```

Comment:
https://github.com/next-step/kotlin-racingcar/pull/1235#discussion_r1241223462 부분은 반영이 안된 것 같아요. 👀

아래와 같이 변경해볼 수 있을 것 같아요. 😉
```suggestion
                println(CAR_SYMBOL.repeat(moving))
```